tests/ecmp/test_fgnhg.py

```diff
@@ -101,8 +117,17 @@ def generate_fgnhg_config(duthost, ip_to_port, bank_0_port, bank_1_port, prefix)
     duthost.shell("sonic-cfggen -j /tmp/fgnhg.json --write-to-db")
 
 
+def setup_neighbors_from_minigraph(duthost):
+    mg_facts = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
```

Comment:
Does duthost.minigraph_facts call into the switch to get the minigraph facts? If that is the case, I would recommend doing a single call of duthost.minigraph_facts instead of multiple in setup_neighbors and generate_fgnhg. This should help reduce test time and improve performance of test.

Reply:
This function is removed